frontend/components/create-recipe-types/create-recipe-helper.tsx

```diff
@@ -0,0 +1,101 @@
+// type declarations and data for create-recipe page
+export type Recipe = {
+  name: string;
+  mins_prep: string;
+  mins_cook: string;
+  description: string;
+  default_servings: string;
+  procedure: string;
+  created_at: string;
+  author_id: string;
+  tags: { key: string; value: string }[];
+  ingredients: {
+    quantity: string;
+    unit: string;
+    name: string;
+    storage: string;
+  }[];
+}
+
+export type FormattedRecipe = {
+  name: string;
+  mins_prep: number;
+  mins_cook: number;
+  description: string;
+  default_servings: number;
+  procedure: string;
+  created_at: string;
+  author_id: string | null;
+  tags: { key: string; value: string }[];
+  ingredients: {
+    quantity: number;
+    unit: string;
+    name: string;
+    storage: string;
+  }[];
+};
+
+// Empty Form Data
+export const emptyData = {
+  name: "",
+  mins_prep: "",
+  mins_cook: "",
+  description: "",
+  default_servings: "",
+  procedure: "",
+  created_at: "",
+  author_id: "",
+  tags: [
+    {
+      key: "",
+      value: "",
+    }
+  ],
+  ingredients: [
+    {
+      name: "",
+      storage: "",
+      quantity: "",
+      unit: "",
+    }
+  ]
+};
+
+// EXAMPLE DATA - used in "Fill Form - Test" button
+export const sampleData = {
```

Comment:
Very "conscientious" of you to have sample data for testing purposes

Reply:
One of the things I focused on for this sprint was making my code readable and more easily testable, which included adding the sample data. Thanks!